app/polymer/package.json

```diff
@@ -33,6 +33,8 @@
     "autoprefixer": "^7.1.6",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
+    "babel-plugin-macros": "^2.0.0",
+    "babel-plugin-paths": "^1.0.0",
```

Comment:
As I understand `babel-plugin-paths` is needed only in the example app, why should it be added to every `app/*` ?